packages/jest/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@metamask/eslint-config-jest",
+  "version": "5.0.0",
+  "description": "Shareable MetaMask ESLint config for Jest.",
+  "main": "src/index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "files": [
+    "src/"
+  ],
+  "scripts": {
+    "test": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MetaMask/eslint-config.git"
+  },
+  "author": "Whymarrh Whitby <whymarrh.whitby@gmail.com>",
```

Comment:
Nit: we should probably omit the `author` field. We omit it in most packages

Reply:
4972ea2